src/styles/style.css

```diff
@@ -260,10 +274,26 @@ body {
 	padding: inherit;
 }
 
+.invest-sidebar-col .alert {
+	display: flex;
+	flex-direction: column;
```

Comment:
Oh cool. I really should use `flex` styling more

Reply:
I don't know. It's baked into bootstrap. It is important to respond to changing window sizes, but this stuff is so "mobile-first" or at mobile-ready that it might not have been the best choice in hindsight.